src/addon.cpp

```diff
@@ -0,0 +1,20 @@
+#include <napi.h>
```

Comment:
Can we move this to a folder named "addon" and maybe name this "zstd.cpp" aligning with mongodb-client-encryption?

Reply:
Done.